src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -30,152 +46,144 @@ static void beforeAll() {
     @BeforeEach
     void setUp() {
         user = new User();
-        user.setAge(DEFAULT_VALID_AGE);
-        user.setLogin(DEFAULT_VALID_LOGIN);
-        user.setPassword(DEFAULT_VALID_PASSWORD);
+        user.setAge(RegistrationServiceTestConstants.DEFAULT_VALID_AGE);

```

Comment:
Not a mistake, but you can use static imports for the constants to make the code more concise and readable.